src/main/java/com/jagrosh/jmusicbot/commands/dj/QueueTypeCmd.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
QueueTypeCmd feels more like an admin command (modifying a server-wide setting) rather than a dj command (modifying the existing playback)

Reply:
makes sense, i moved it 👾